tests/test_variational.py

```diff
@@ -31,6 +32,10 @@ def model():
     "ADVI": {
         "method": pm.MeanField, 
         "fit_kwargs": {}
+    },
```

Comment:
we need tests to make sure all making sense kwargs work properly (`sample_size`, etc)

Reply:
Resolved in ccd626c.